src/app/components/elements/panels/Banner.tsx

```diff
@@ -7,23 +7,39 @@ export interface BannerProps {
   subtitle: string;
   link: string;
   src: string;
+  secondary?: boolean;
   alt?: string;
 }
 
-export const Banner = ({ id, title, subtitle, link, src, alt, children }: PropsWithChildren<BannerProps>) => {
+export const Banner = ({
+  id,
+  title,
+  subtitle,
+  link,
+  src,
+  secondary,
+  alt,
+  children,
+}: PropsWithChildren<BannerProps>) => {
   return (
-    <section id={id} className="bg-primary pattern-05 p-5">
+    <section
+      id={id}
+      className={`${secondary ? "text-white" : "bg-primary"} pattern-05 p-5`}
+      style={secondary ? { backgroundColor: "#07174f" } : {}}
+    >
       <Container>
         <Row className="d-flex align-items-center justify-content-center">
-          <button className="button-header btn-lg">{title}</button>
+          <button className="btn-lg" style={secondary ? { backgroundColor: "white" } : { backgroundColor: "black" }}>
```

Comment:
It may be a bit more readable to move all these `secondary ? ...` to their own variables (4 in total). That way there is less noise when reading all this JSX. For instance, this line would be read as:
```typescript
<button className="btn-lg" style={buttonStyle}>
```
Also, this particular one can be rewritten as:
```typescript
{ backgroundColor: secondary ? "white" : "black" }
```

Reply:
I've changed this now so it's a single custom class being added (`banner-primary` or `banner-secondary`) that adjusts all of the required colours.